backend/src/v5/models/tickets.logs.js

```diff
@@ -59,4 +59,13 @@ TicketLogs.addGroupUpdateLog = async (teamspace, project, model, ticket, groupId
 TicketLogs.addTicketLog = (teamspace, project, model, ticket, ticketLog) => DB.insertOne(teamspace,
 	TICKET_LOGS_COL, { ...ticketLog, _id: generateUUID(), teamspace, project, model, ticket });
 
+TicketLogs.getTicketLogsById = (
+	teamspace,
+	project,
+	model,
+	ticket,
+	projection = { _id: 0, author: 1, changes: 1, timestamp: 1 },
+	sort = { timestamp: 1 },
+) => DB.find(teamspace, TICKET_LOGS_COL, { teamspace, project, model, ticket }, projection, sort);
```

Comment:
Actually apart from the changes we need to fetch "imported" property. When we import a ticket via the ticket import endpoint we add a log that has no changes but an "imported" property. We have to fetch this too and serialise and return it it as we do with changes. 

If you can, import a ticket and make sure we cover this scenario as well. Also if you can add a few tests that cover this too.